cumulusci/cumulusci.yml

```
@@ -252,6 +252,10 @@ tasks:
         description: Retrieves a list of the currently available Permission Set License definition keys
         class_path: cumulusci.tasks.preflight.licenses.GetAvailablePermissionSetLicenses
         group: Salesforce Preflight Checks
+    get_assigned_permission_sets:
+        description: Retrieves a list of the label and names of any permission sets assigned to the running user.
```

Sorry, one more proposed tweak:

```suggestion
        description: Retrieves a list of the names of any permission sets assigned to the running user.
```